tests/integration-tests/tests/storage/test_ebs.py

```diff
@@ -45,7 +45,7 @@ def test_ebs_snapshot(
 ):
     logging.info("Testing ebs snapshot")
     mount_dir = "ebs_mount_dir"
-    volume_size = 10
+    volume_size = 21
```

Comment:
Can you add a comment above this line explaining why you chose this value?

Reply:
Yes, done!